_posts/news/2021-04-01-announcing-scalajs-1.5.1.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+title: Announcing Scala.js 1.5.1
+category: news
+tags: [releases]
+permalink: /news/2020/11/16/announcing-scalajs-1.5.1/
+---
+
+
+We are pleased to announce the release of Scala.js 1.5.1!
+
+This release mostly contains additions and improvements in the JDK libraries.
+In particular, several APIs involving floating point numbers have been fixed, from parsing to bit manipulation to formatting.
+
+Read on for more details.
+
+<!--more-->
+
+## Getting started
+
+If you are new to Scala.js, head over to [the tutorial]({{ BASE_PATH }}/tutorial/).
+
+If you need help with anything related to Scala.js, you may find our community [on Gitter](https://gitter.im/scala-js/scala-js) and [on Stack Overflow](https://stackoverflow.com/questions/tagged/scala.js).
+
+Bug reports can be filed [on GitHub](https://github.com/scala-js/scala-js/issues).
+
+## Release notes
+
+If upgrading from Scala.js 0.6.x, make sure to read [the release notes of Scala.js 1.0.0]({{ BASE_PATH }}/news/2020/02/25/announcing-scalajs-1.0.0/) first, as they contain a host of important information, including breaking changes.
+
+This is a **patch** release:
+
+* It is backward binary compatible with all earlier versions in the 1.x series: libraries compiled with 1.0.x through 1.5.0 can be used with 1.5.1 without change.
+* It is forward binary compatible with 1.5.0: libraries compiled with 1.5.1 can be used with 1.5.0 without change.
+* It is backward source compatible with 1.5.0: source code that used to compile with 1.5.0 should compile as is when upgrading to 1.5.1.
+
+From Scala.js 1.5.0, which was a **minor** release:
+
+* It is *not* forward binary compatible with 1.4.x: libraries compiled with 1.5.1 cannot be used with 1.4.x or earlier.
+* It is *not* entirely backward source compatible: it is not guaranteed that a codebase will compile *as is* when upgrading from 1.4.x or earlier (in particular in the presence of `-Xfatal-warnings`).
```

Comment:
If you put "In addition, like Scala.js 1.5.0:", maybe add "... backward source compatible with 1.4.x:" to be more explicit.

Reply:
Good point :)